### PR TITLE
Feat/last5yr graph

### PR DESCRIPTION
**Pull request by AKhoo**

### CODE REVIEW

**Review by ccades — APPROVED** [review]
lgtm! Everything seems neat and correct so as long as it runs for you I'd say all good.

File: client/components/app.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+import Chart5y from './chart5y.jsx';
```
Comment (therobinkim) [inline client/components/app.jsx:4]: capitalize the `Chart5y.jsx` file name (as per Airbnb)

also, there's no need to name files `.jsx`; would advise `.js` instead (this is where I diverge from Airbnb, but you can choose to stick w/ their guidelines)

https://github.com/airbnb/javascript/tree/master/react#naming

File: client/components/app.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  render() {
+    var tooltipY = 110;
+    var change = (this.state.selectedPrice - this.state.refStartPrice).toFixed(2);
+    var changePercent = ((this.state.selectedPrice - this.state.refStartPrice) * 100 / this.state.refStartPrice).toFixed(2);
+    return (
+      <div>
+        <CompanyName>{this.state.companyName}</CompanyName>
+        <div><Odometer value={this.state.selectedPrice} format='(,ddd).dd' duration={300}></Odometer></div>
+        <div><Change>{change > 0 ? '+$' : '-$'} {Math.abs(change)} {'(' + changePercent + '%) '}</Change><ChangeCaption>{this.state.changeCaption}</ChangeCaption></div>
+        <Chart5y requestData={this.requestData} setSelectedPrice={this.setSelectedPrice} handleMouseLeaveChart={this.handleMouseLeaveChart} setChangeCaption={this.setChangeCaption} tooltipY={tooltipY}/>
```
Comment (therobinkim) [inline client/components/app.jsx:117]: would advise breaking this long line up https://github.com/airbnb/javascript/tree/master/react#alignment

File: client/components/chart5y.jsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (therobinkim) [inline client/components/chart5y.jsx:2]: doesn't seem like `react-dom` is used anywhere in this file; should probably not `import` it

File: client/components/chart5y.jsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        return [item.datetime, item.price];
+      });
+
+      this.chart.series[0].setData(data);
+    });
+  }
+
+  componentDidMount() {
+    var setSelectedPrice = this.props.setSelectedPrice;
+    var setChangeCaption = this.props.setChangeCaption;
```
Comment (therobinkim) [inline client/components/chart5y.jsx:29]: if you wanted you could take advantage of destructuring
```
var {
  setChangeCaption,
  setSelectedPrice
} = this.props;
```

File: client/components/chart5y.jsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+      yAxis: {
+        visible: false
+      }
+    });
+  }
+
+  render() {
+    return (
+      <div id='container' onMouseLeave={() => {this.props.handleMouseLeaveChart(defaultCaption)}}>
```
Comment (therobinkim) [inline client/components/chart5y.jsx:118]: use double quotes in jsx
(would advise using Airbnb style guide, not `eslint-config-hackreactor`)
https://github.com/airbnb/javascript/tree/master/react#quotes

File: public/bundle.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...refix":kt=null,i?"function"!=typeof i?gt=1:(gt=2,kt=i):gt=0}}return $t}function Vt(e,i){if(void 0!==this&&this.constructor===Vt)return t(e);var r=e,o=r.charCodeAt(0);o<33&&(o=(r=r.trim()).charCodeAt(0)),_t>0&&(Dt=r.replace(p,o===R?"":"-")),o=1,1===vt?Rt=r:zt=r;var a,s=[Rt];Mt>0&&void 0!==(a=Bt(Pt,i,s,s,mt,ft,0,0,0,0))&&"string"==typeof a&&(i=a);var l=Ft(Tt,s,i,0,0);return Mt>0&&void 0!==(a=Bt(Et,l,s,s,mt,ft,l.length,0,0,0))&&"string"!=typeof(l=a)&&(o=0),Dt="",Rt="",zt="",yt=0,mt=1,ft=1,wt*o==0?l:l.replace(n,"").replace(v,"").replace(g,"$1").replace(x,"$1").replace(w," ")}return Vt.use=function t(e){switch(e){case void 0:case null:Mt=Ct.length=0;break;default:if("function"==typeof e)Ct[Mt++]=e;else if("object"==typeof e)for(var i=0,n=e.length;i<n;++i)t(e[i]);else Nt=0|!!e}return t},Vt.set=$t,void 0!==e&&$t(e),Vt}(null)},function(t,e,i){"use strict";var n=function(t,e){return t===e};e.a=function(t,e){var i;void 0===e&&(e=n);var r,o=[],a=!1,s=function(t,i){return e(t,o[i])};return function(){for(var e=arguments.length,n=new Array(e),l=0;l<e;l++)n[l]=arguments[l];return a&&i===this&&n.length===o.length&&n.every(s)?r:(r=t.apply(this,n),a=!0,i=this,o=n,r)}}},function(t,e,i){"use strict";
```
Comment (therobinkim) [inline public/bundle.js:1]: don't commit your bundle.js

File: webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx$/,
+        exclude: /(node_modules|bower_components)/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            plugins: ["babel-plugin-styled-components"],
```
Comment (therobinkim) [inline webpack.config.js:17]: use single quotes, not double quotes, bc you use single quotes everywhere else

File: webpack.config.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  entry: './client/components/app.jsx',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'public')
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx$/,
+        exclude: /(node_modules|bower_components)/,
```
Comment (therobinkim) [inline webpack.config.js:13]: are you using `bower`?

**Pull request merged**